internal/model/transaction.go

```diff
@@ -115,8 +112,8 @@ func (e *Transaction) fields() mapstr.M {
 	if e.FailureCount != 0 {
 		transaction.set("failure_count", e.FailureCount)
 	}
-	if e.SuccessCount != 0 {
-		transaction.set("success_count", e.SuccessCount)
+	if e.SuccessCount.Count != 0 {
```

Comment:
Should success_count be omitted from output if Count is zero? e.g. when it only contains unknown outcome transactions.

Reply:
If it is not omitted on count=0, `transaction.success_count` will have to be a pointer to SummaryMetric, otherwise any event that contains the transaction field will have `transaction.success_count.sum: 0` and `transaction.success_count.count: 0`